app/src/main/java/fr/free/nrw/commons/mwapi/ApacheHttpClientMediaWikiApi.java

```diff
@@ -54,6 +49,26 @@ public ApacheHttpClientMediaWikiApi(String apiURL) {
         api = new CustomMwApi(apiURL, httpClient);
     }
 
+    @Override
+    public String getEditToken() throws IOException {
+        String editToken = api.action("query")
+                .param("meta", "tokens")
+                .post()
+                .getString("/api/query/tokens/@csrftoken");
+        Timber.d("MediaWiki edit token is %s", editToken);
+        return editToken;
+    }
+
+
+    @Override
+    public String getFileEntityId(String fileName) throws IOException {
```

Comment:
JavaDocs

Reply:
Ashish I have not added this function. I have added a relevant java docs to it. However I see that this function isn't being used anywhere, Is there any use of this function in future or shall I remove it?

Reply:
If you have not added it and it does not exist in the base branch, we can safely remove this as long as the build does not breaks